anaDACScan.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+"""
+anaDACScan
+==============
+"""
```

Comment:
You should add a help string here; this will help `sphinx` automatically build the html documentation for the repo, see for example:

https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/develop/packageFiles4Docker.py#L3-L98

Reply:
done